pythoncalculator/multiply.py

```diff
@@ -0,0 +1,2 @@
+def multiply(x, y):
+    return x + y
```

Comment:
I think there is a typo here. Probably you wanted to return a product rather than a sum.